### PR TITLE
Verbose bool

### PR DESCRIPTION
**Pull request by MarioOcampo**
Implementation of the requested change #90: the `verbose` parameters were changed to `boolean`. 

An additional change in `trk2dictionary`. 
Now it is checked that the tractogram `exists` before trying to load it, and the section `try` was removed.

### CODE REVIEW

File: commit/trk2dictionary/trk2dictionary.pyx [hunk 1]
```diff
@@ -250,13 +250,14 @@ cpdef run( filename_tractogram=None, path_out=None, filename_peaks=None, filenam
     # Streamlines from tractogram
     print( '\t- Tractogram' )
     
+    if not exists(filename_tractogram):
+        ERROR( 'Tractogram file not found: %s' % filename_tractogram )
```
Comment (daducci) [inline commit/trk2dictionary/trk2dictionary.pyx:254]: Could you add also a line in the CHANGELOG to keep track that we "fixed" this?

Comment (MarioOcampo) [reply commit/trk2dictionary/trk2dictionary.pyx:254]: Yes

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ All notable changes to COMMIT will be documented in this file.
 ### Changed
 - core.pyx: The function build_operator checks if the LUT configuration 
             changed before build a new operator
+- trk2dictionary.pyx: Check that the tractogram exists before trying to
```
Comment (daducci) [inline CHANGELOG.md:16]: Put this under "Fixed" to highlight that this modification is a fix

Comment (MarioOcampo) [reply CHANGELOG.md]: OK, done

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,12 +2,14 @@
...
 
-## [1.4.5] - 2020-12-29
+## [1.4.5] - 2021-02-08
 
 ### Fixed
 - operator.pyxbld: Changed the condition to create a new operator
 - trk2dictionary.pyx: Check that the tractogram exists before trying to 
             load it and remove the try section
+- trk2dictionary.run(): fixed bug in computing the blur
+- trk2dictionary.run(): fixed bug with blur parameters
```
Comment (daducci) [inline CHANGELOG.md:12]: Can you merge these two lies into one (they refer to the same fix)

Comment (MarioOcampo) [reply CHANGELOG.md]: Yes, done

**Pull request merged**